netbout-web/src/test/java/com/netbout/rest/TkPreviewTest.java

```diff
@@ -77,7 +77,7 @@ public void previewMessageBeforePost() throws Exception {
                 )
             ).printBody(),
             Matchers.allOf(
-                Matchers.containsString("<p>Here is a paragraph.</p>"),
+                Matchers.containsString("<p>Here is a paragraph.<br  /></p>"),
```

Comment:
@carlosmiranda just created an issue for that https://github.com/jcabi/jcabi-matchers/issues/29